docs/references/api/deviceshifu-configmap.md

```diff
@@ -36,19 +36,20 @@ DeviceShifuInstructions are all the commands and settings that ***deviceShifu***
 - **instructionSettings** ([DeviceShifuInstructionSettings](#deviceshifuinstructionsettings))
 - **instructions** (map[string]DeviceShifuInstruction)
  **Example:**
+
```

Comment:
remove extra line, please double check the entire PR make sure there are no extra changes.